Client/Frontend/Browser/ButtonToast.swift

```diff
@@ -39,7 +39,7 @@ class ButtonToast: UIView {
     }()
     fileprivate var animationConstraint: Constraint?
     fileprivate lazy var gestureRecognizer: UITapGestureRecognizer = {
-        let gestureRecognizer = UITapGestureRecognizer(target: self, action: #selector(ButtonToast.handleTap(_:)))
+        let gestureRecognizer = UITapGestureRecognizer(target: self, action: #selector(ButtonToast.handleTap))
```

Comment:
Nit: why leave `ButtonToast` here?

Reply:
Those where I just remove (_:) were done with a search and replace. I will remove the class names in the next PR. This one was getting a little bloated.